groovy/groovy.editor/src/org/netbeans/modules/groovy/editor/api/parser/GroovyVirtualSourceProvider.java

```diff
@@ -350,7 +350,8 @@ private void genProp(PropertyNode propNode, PrintWriter out) {
             boolean skipGetter = false;
             List getterCandidates = propNode.getField().getOwner().getMethods(getterName);
             if (getterCandidates != null) {
-                for (MethodNode method : getterCandidates) {
```

Comment:
Are you looking for this?

```diff
--- a/groovy/groovy.editor/src/org/netbeans/modules/groovy/editor/api/parser/GroovyVirtualSourceProvider.java
+++ b/groovy/groovy.editor/src/org/netbeans/modules/groovy/editor/api/parser/GroovyVirtualSourceProvider.java
@@ -348,10 +348,9 @@
             String getterName = "get" + name;
 
             boolean skipGetter = false;
-            List getterCandidates = propNode.getField().getOwner().getMethods(getterName);
+            List<MethodNode> getterCandidates = propNode.getField().getOwner().getMethods(getterName);
             if (getterCandidates != null) {
-                for (Iterator it = getterCandidates.iterator(); it.hasNext();) {
-                    MethodNode method = (MethodNode) it.next();
+                for (MethodNode method: getterCandidates) {
                     if (method.getParameters().length == 0) {
                         skipGetter = true;
                     }
```

Reply:
Yes.. It was an escape for something I was testing..

Should be reverted..

Good eyes, BTW..